README.md

```diff
@@ -95,125 +123,27 @@ query CityById{
 }
 ```
 
----
-# README
-
-Welcome to [RedwoodJS](https://redwoodjs.com)!
-
-> **Prerequisites**
->
-> - Redwood requires [Node.js](https://nodejs.org/en/) (>=14.19.x <=16.x) and [Yarn](https://yarnpkg.com/) (>=1.15)
-> - Are you on Windows? For best results, follow our [Windows development setup](https://redwoodjs.com/docs/how-to/windows-development-setup) guide
+## Running Edge Functions in GraphQL
 
-Start by installing dependencies:
+Add the following to your `netlify.toml` file:
 
 ```
-yarn install
+[[edge_functions]]
+path = "/.redwood/functions/graphql"
+function = "geolocate"
 ```
 
-Then change into that directory and start the development server:
+Adding this to the `.toml` file means that when a request is made to the `path`, it will invoke the `geolocate` edge function as part of the request.
 
-```
-cd my-redwood-project
-yarn redwood dev
-```
-
-Your browser should automatically open to http://localhost:8910 where you'll see the Welcome Page, which links out to a ton of great resources.
-
-> **The Redwood CLI**
->
-> Congratulations on running your first Redwood CLI command!
-> From dev to deploy, the CLI is with you the whole way.
-> And there's quite a few commands at your disposal:
-> ```
-> yarn redwood --help
-> ```
-> For all the details, see the [CLI reference](https://redwoodjs.com/docs/cli-commands).
-
-## Prisma and the database
-
-Redwood wouldn't be a full-stack framework without a database. It all starts with the schema. Open the [`schema.prisma`](api/db/schema.prisma) file in `api/db` and replace the `UserExample` model with the following `Post` model:
-
-```
-model Post {
-  id        Int      @id @default(autoincrement())
-  title     String
-  body      String
-  createdAt DateTime @default(now())
-}
-```
+## Testing Edge Functions
 
-Redwood uses [Prisma](https://www.prisma.io/), a next-gen Node.js and TypeScript ORM, to talk to the database. Prisma's schema offers a declarative way of defining your app's data models. And Prisma [Migrate](https://www.prisma.io/migrate) uses that schema to make database migrations hassle-free:
+1. Place edge function in `netlify/edge-functions`
+2. In `netlify.toml`, your path is to your serverless function. In order to declare that the edge function run for that path, you specify the function name as the `function` value. Be sure to include the deploy path where functions live, like `/.redwood/functions/` or `/.netlify/functions/`
```

Comment:
[pebble] Breaking this up a bit so that it's easier to follow.
```suggestion
2. In `netlify.toml`, set the `path` value to your serverless function's path. 
  Be sure to include the deploy path where functions live, like `/.redwood/functions/` or `/.netlify/functions/`.
3. Set the `function` value to the name of your Edge Function.
```